assets/component-predictive-search.css

```
@@ -50,8 +65,40 @@ predictive-search[loading] .predictive-search {
   display: block;
 }
 
+.predictive-search__result-group {
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 auto;
+  gap: 3rem;
```

```suggestion
  gap: 2rem;
```
I find the 3rem gap to be very large. Suggesting 2rem to balance white space. Thoughts @al-mcquiston ?

---

@melissaperreault totally agree. myself and @FCalabria chatted about this and this should be updated to your suggestion